npm/cmd/start.go

```diff
@@ -58,36 +60,48 @@ func newStartNPMCmd() *cobra.Command {
 			config := &npmconfig.Config{}
 			err := viper.Unmarshal(config)
 			if err != nil {
-				return fmt.Errorf("failed to load config with error %w", err)
+				return fmt.Errorf("failed to load config with error: %w", err)
+			}
+
+			flags := npmconfig.Flags{
+				KubeConfigPath: viper.GetString(FlagKubeConfigPath),
 			}
 
-			return start(*config)
+			return start(*config, flags)
 		},
 	}
+
+	startNPMCmd.Flags().String(FlagKubeConfigPath, FlagDefaults[FlagKubeConfigPath], "path to kubeconfig")
+
 	return startNPMCmd
 }
 
-func start(config npmconfig.Config) error {
+func start(config npmconfig.Config, flags npmconfig.Flags) error {
 	klog.Infof("loaded config: %+v", config)
 	klog.Infof("Start NPM version: %s", version)
 
 	var err error
-	defer func() {
```

Comment:
Is there a reason to remove recover ?

Reply:
if we have a crash we want to crash hard and bubble that up to orchestrator